From 2702b452d8afd621d055351de164aa00ed07546c Mon Sep 17 00:00:00 2001
From: Ales Novak <alnovak@suse.cz>
Date: Mon, 28 May 2018 16:24:02 +0200
References: bsc#1094823
Patch-mainline: never, SUSE specific
Subject: [PATCH] Fix excessive newline in /proc/*/status.

In upstream, "Seccomp:*" line is printed using seq_put_decimal_ull(),
which doesn't append newline, hence it is added in
"\nSpeculation_Store_Bypass:".

This excessive newline is puzzling some programs like iotop.

Signed-off-by: Ales Novak <alnovak@suse.cz>
---
 fs/proc/array.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/proc/array.c b/fs/proc/array.c
index cb71cbae606d..60cbaa821164 100644
--- a/fs/proc/array.c
+++ b/fs/proc/array.c
@@ -333,7 +333,7 @@ static inline void task_seccomp(struct seq_file *m, struct task_struct *p)
 #ifdef CONFIG_SECCOMP
 	seq_printf(m, "Seccomp:\t%d\n", p->seccomp.mode);
 #endif
-	seq_printf(m, "\nSpeculation_Store_Bypass:\t");
+	seq_printf(m, "Speculation_Store_Bypass:\t");
 	switch (arch_prctl_spec_ctrl_get(p, PR_SPEC_STORE_BYPASS)) {
 	case -EINVAL:
 		seq_printf(m, "unknown");
-- 
2.11.0.258.ge05806d

